### PR TITLE
Orderbook reserve: lister changes to connect to FeeBurner

### PR DESCRIPTION
**Pull request by amitkot**

### CODE REVIEW

**Comment by amitkot** [comment]
@ilanDoron

File: contracts/permissionless/PermissionlessOrderBookReserveLister.sol [hunk 1]
```diff
@@ -24,6 +37,10 @@ contract PermissionlessOrderBookReserveLister {
     mapping(address => ListingStage) public reserveListingStage;
     mapping(address => OrderBookReserveInterface) public reserves;
 
+    // KNC burn fee per order that is taken. = 25 / 1000 = 0.25 %
+    // TODO: pass this to OrderBookReserve's ctor
```
Comment (ilanDoron) [inline contracts/permissionless/PermissionlessOrderBookReserveLister.sol]: why don't u pass it in Ctor?
u can do it!

File: test/permissionlessOrderBookReserveLister.js [hunk 2]
```diff
@@ -260,22 +277,133 @@ contract('PermissionlessOrderBookReserveLister', async (accounts) => {
...
-    await KNCToken.approve(res.address, kncTwei);
-    await res.makerDepositKnc(maker, kncTwei);
-    await res.makerDepositWei(maker, {from: maker, value: ethWei});
+    await res.depositToken(maker, tokenTwei);
+    await kncToken.approve(res.address, kncTwei);
+    await res.depositKncFee(maker, kncTwei);
+    await res.depositEther(maker, {from: maker, value: ethWei});
 }
 
 async function twoStringsSoliditySha(str1, str2) {
```
Comment (ilanDoron) [inline test/permissionlessOrderBookReserveLister.js]: seems can remove this function...
I guess I put it there. but if u make changes. remove it.

File: contracts/permissionless/PermissionlessOrderBookReserveLister.sol (hunk 1 above)
Comment (amitkot) [reply contracts/permissionless/PermissionlessOrderBookReserveLister.sol]: Done

File: test/permissionlessOrderBookReserveLister.js (hunk 2 above)
Comment (amitkot) [reply test/permissionlessOrderBookReserveLister.js]: Removed unused functions.

**Pull request merged**